src/Cart.js

```diff
@@ -0,0 +1,38 @@
+import initialStoreItems from './store-items'
+console.log(initialStoreItems)
+
```

Comment:
this is  nice because this whole component handles the specific section it is rendering